apps/dashboard/app/apps/nav_bar.rb

```diff
@@ -0,0 +1,103 @@
+class NavBar
+
+  attr_reader :nav_items
+
+  def self.from_config(nav_bar)
+    nav_items = nav_bar.map do |nav_item|
+      NavBar.parse_nav_item(nav_item)
+    end
+
+    NavBar.new(nav_items: nav_items)
+  end
+
+  def initialize(nav_items: [])
+    @nav_items = nav_items
+  end
+
+  def has_items?
+    nav_items.size > 0
+  end
```

Comment:
```suggestion
  def empty?
    nav_items.size == 0
  end
```

There's also a private `items?` below. Seems like all of these can coalesce to 1 API.  In any case, with the `?` we don't typically need `has` or `is` prefixes.

Reply:
Will update